index.html

```diff
@@ -2107,23 +2115,50 @@ <h2>Definition of Roles</h2>
+          Typically, the initial state of a <code>combobox</code> is collapsed.
+          In the collapsed state, only the <code>combobox</code> element and an optional popup control <rref>button</rref> are visible.
+          A <code>combobox</code> is said to be expanded when both the <code>combobox</code> element showing its current value and its associated popup element are visible.
+          Authors MUST set <sref>aria-expanded</sref> to <code>true</code> on an element with role <code>combobox</code> when it is expanded and <code>false</code> when it is collapsed.
+        </p>
+				<p>
+          Authors MUST ensure the popup element associated with a <code>combobox</code> has a role of <rref>listbox</rref>, <rref>tree</rref>, <rref>grid</rref>, or <rref>dialog</rref>.
+          Authors MUST set <pref>aria-controls</pref> on a <code>combobox</code> element to a value that refers to the <code>combobox</code> popup element.
+        </p>
+				<p>
+          Elements with the role <code>combobox</code> have an implicit <pref>aria-haspopup</pref> value of <code>listbox</code>.
+          If the <code>combobox</code> popup element has a role other than <rref>listbox</rref>, authors MUST specify a value for <pref>aria-haspopup</pref> that corresponds to the type of its popup.
+        </p>
+				<p>
+          If the user interface includes an additional icon that allows the visibility of the popup to be controlled via pointer and touch events, authors SHOULD ensure that element has role <rref>button</rref> and that it is focusable but not included in the page <kbd>Tab</kbd> sequence.
+          In addition, to be keyboard accessible, authors SHOULD provide keyboard mechanisms for moving focus between the <code>combobox</code> element and elements contained in the popup.
+          For example, one common convention is that <kbd>Down Arrow</kbd> moves focus from the input to the first focusable descendant of the popup element.
+          If the popup element supports <pref>aria-activedescendant</pref>, in lieu of moving focus, such keyboard mechanisms can control the value of <pref>aria-activedescendant</pref> on the <code>combobox</code> element.
+          When a descendant of the popup element is active, authors MAY set <pref>aria-activedescendant</pref> on the <code>combobox</code> to a value that refers to the active element within the popup while focus remains on the <code>combobox</code> element.
+        </p>
 				<pre class="example highlight">
-				  &lt;div aria-label="Tag" role="combobox" aria-expanded="true" aria-owns="owned_listbox" aria-haspopup="listbox"&gt;
-				      &lt;input type="text" aria-autocomplete="list" aria-controls="owned_listbox" aria-activedescendant="selected_option"&gt;
-				  &lt;/div&gt;
-				  &lt;ul role="listbox" id="owned_listbox"&gt;
-				      &lt;li role="option"&gt;Zebra&lt;/li&gt;
-				      &lt;li role="option" id="selected_option"&gt;Zoom&lt;/li&gt;
+		      &lt;input type="text" aria-label="Tag"
```

Comment:
This example would be friendlier if it used a `<label for="the-text-input">Tag</label>` instead of an invisible `aria-label="Tag"`. :)

Reply:
Curious why not? As a sighted user, I prefer when a combobox has a visible label. I think we should model the most inclusive behavior?

Reply:
That was supposed to be a thumbs up ... guess I somehow chose wrong emoji. Making the change.